packages/block-editor/src/components/block-settings-menu-controls/index.js

```diff
@@ -46,7 +49,7 @@ const BlockSettingsMenuControlsSlot = ( { fillProps, clientIds = null } ) => {
 	const { isGroupable, isUngroupable } = convertToGroupButtonProps;
 	const showConvertToGroupButton = isGroupable || isUngroupable;
 	return (
-		<Slot fillProps={ { ...fillProps, selectedBlocks } }>
+		<Slot fillProps={ { ...fillProps, selectedBlocks, selectedClientIds } }>
```

Comment:
This is a new API. Currently we only pass a list of block names to `BlockSettingsMenuControls`. With this change we pass both a list of block names and a list of block client IDs. This lets users of `BlockSettingsMenuControls` know which blocks the block settings menu is adjusting.

I think it would be nicer still if we simply passed a list of block objects but not sure how to do that in a backwards compatible way given that the name `selectedBlocks` is taken. Open to ideas.